tests/check/test_sema_check_cli.py

```
@@ -0,0 +1,25 @@
+def test_main():
+    log.info(f"test_main_check")
+
+    input_folder = Path(__file__).parent / "test_files"
+    output_formats = ["csv", "html", "yml"]
+    for output in output_formats:
+        cli_line = f"--input_folder {input_folder} --output {output}"
+        # Backup the original sys.argv
+        original_argv = sys.argv
+        try:
+            # Set sys.argv to simulate command-line arguments
+            sys.argv = ["sema-check"] + cli_line.split()
```

Ruff

Consider `["sema-check", *cli_line.split()]` instead of concatenation

Replace with `["sema-check", *cli_line.split()]`

(RUF005)